app/models/service_template.rb

```diff
@@ -400,8 +400,8 @@ def self.create_from_options(options)
   end
   private_class_method :create_from_options
 
-  def provision_request(user, options = nil)
-    provision_workflow(user, options).submit_request
+  def provision_request(user, options = nil, request_options = nil)
```

Comment:
I understand. It is not a simple get (read only), it actually makes a request. Since it was introduced not long ago, we can refactor it. It is my recommendation, not a have_to_fix issue.